File: Demo/src/main/java/com/aigestudio/wheelpicker/demo/PreviewActivity.java
package com.aigestudio.wheelpicker.demo;

import android.app.Activity;
import android.os.Bundle;
import android.widget.Toast;

import com.aigestudio.wheelpicker.WheelPicker;

/**
 * @author AigeStudio 2015-12-06
 * @author AigeStudio 2016-07-08
 */
public class PreviewActivity extends Activity implements WheelPicker.OnItemSelectedListener {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.ac_preview);

        WheelPicker wheelLeft = (WheelPicker) findViewById(R.id.main_wheel_left);
        wheelLeft.setOnItemSelectedListener(this);
        WheelPicker wheelCenter = (WheelPicker) findViewById(R.id.main_wheel_center);
        wheelCenter.setOnItemSelectedListener(this);
        WheelPicker wheelRight = (WheelPicker) findViewById(R.id.main_wheel_right);
        wheelRight.setOnItemSelectedListener(this);
    }

    @Override
    public void onItemSelected(WheelPicker picker, Object data, int position) {
        String text = "";
        int id = picker.getId();
        if (id == R.id.main_wheel_left) {
            text = "Left:";
        } else if (id == R.id.main_wheel_center) {
            text = "Center:";
        } else if (id == R.id.main_wheel_right) {
            text = "Right:";
        }
        Toast.makeText(this, text + String.valueOf(data), Toast.LENGTH_SHORT).show();
    }
}

File: Demo/src/main/java/com/aigestudio/wheelpicker/demo/PreviewActivity.kt
package com.aigestudio.wheelpicker.demo

import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.aigestudio.wheelpicker.WheelPicker
import com.aigestudio.wheelpicker.demo.databinding.AcPreviewBinding

/**
 * @author AigeStudio 2015-12-06
 * @author AigeStudio 2016-07-08
 */
class PreviewActivity : AppCompatActivity(), WheelPicker.OnItemSelectedListener {
    private lateinit var binding: AcPreviewBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = AcPreviewBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.mainWheelLeft.setOnItemSelectedListener(this)
        binding.mainWheelCenter.setOnItemSelectedListener(this)
        binding.mainWheelRight.setOnItemSelectedListener(this)
    }

    override fun onItemSelected(picker: WheelPicker, data: Any?, position: Int) {
        val text = when (picker.id) {
            binding.mainWheelLeft.id -> "Left:"
            binding.mainWheelCenter.id -> "Center:"
            binding.mainWheelRight.id -> "Right:"
            else -> ""
        }
        Toast.makeText(this, text + data.toString(), Toast.LENGTH_SHORT).show()
    }
}